bindings/python/py_src/safetensors/numpy.py

```diff
@@ -5,12 +5,7 @@
 import numpy as np
 
 from safetensors import deserialize, safe_open, serialize, serialize_file
-
-
-def _tobytes(tensor: np.ndarray) -> bytes:
-    if not _is_little_endian(tensor):
-        tensor = tensor.byteswap(inplace=False)
-    return tensor.tobytes()
+from torch import t
```

Comment:
This adds `torch` as a dependency, we can import torch in code paths that really require Torch.